source/deployment/bulk-loading-data-format.rst

```diff
@@ -125,13 +125,13 @@ For clarity, the object is shown using regular JSON formatting, but in the data
 .. code-block:: javascript
 
   {
-  type: "team",
-  team: {
-    name: "team-name",
-    display_name: "Team Display Name",
-    type: "O",
-    description: "The Team Description",
-    allow_open_invite: true
+  "type": "team",
+  "team": {
+    "name": "team-name",
+    "display_name": "Team Display Name",
+    "type": "O",
+    "description": "The Team Description",
+    "allow_open_invite": "true"
```

Comment:
@grundleborg Should `true` be in quotes or not?

Reply:
None of the values (the stuff to the right of the `:` character) should be changed - they already have quotes if they need them. It's just all the keys that need fixing (the stuff to the left of the `:` character). `true` should *not* be in quotes.